util/testutil/integration/run.go

```diff
@@ -25,17 +25,24 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-type Sandbox interface {
+// Backend is the minimal interface implemented by testing backends that allow
+// us to connect and print their logs.
+type Backend interface {
 	Address() string
 	PrintLogs(*testing.T)
```

Comment:
printlogs is not backend specific, rootless is backend specific

Reply:
Fixed.